api/src/apollo/Event/queries.ts

```diff
@@ -1,42 +1,25 @@
 import { checkGqlPermissions } from 'helpers/auth/checkPermissions';
```

Comment:
Status is needed, because multiple test events would also.be present. In fact this file can be kept unchanged, just need to add `isTeamEvent` nullable parameter.